daemon/src/model/cfd.rs

```diff
@@ -1276,6 +1279,273 @@ impl Cfd {
     }
 }
 
+trait Aggregate {
+    type Item;
+
+    fn version(&self) -> u64;
+    fn apply(self, evt: &Self::Item) -> Self
+    where
+        Self: Sized;
+}
+
+pub mod taker {
+
+    use super::*;
+
+    /// Models the cfd state
+    ///
+    /// Upon `Command`s, that are reaction to something happening in the system, we decide to
```

Comment:
I think this is misleading, a command is not a reaction as such. Yes we are "reacting" to timelocks etc but these commands are the _essence_ of the system. They are active controls.

Reply:
Would you mind adding a suggestion?